src/report/ReportManager.ts

```diff
@@ -693,35 +693,41 @@ class DisplayManager {
             // Ignore.
         }
 
-        let eventContent;
+        enum OutType {
+            msg, text, html
+        }
+
+        let eventContent: [OutType, string];
```

Comment:
Alternatively, this could have been
```ts
let eventContent = { msg: string } | { html: string } | { text: string};
```

which I think would be a bit easier to read.

Your call.